libs/components/progress-indicator/src/lib/modules/progress-indicator/progress-indicator-status-marker/progress-indicator-status-marker.component.ts

```diff
@@ -26,75 +26,44 @@ import { SkyProgressIndicatorItemStatus } from '../types/progress-indicator-item
 export class SkyProgressIndicatorStatusMarkerComponent implements OnDestroy {
   @Input()
   public set displayMode(value: 'vertical' | 'horizontal') {
```

Comment:
Should be `| undefined`?

Reply:
I left off `| undefined` on internal component inputs that are always set to a value.